scripts/player.gd

```diff
@@ -78,11 +110,21 @@ func _physics_process(delta) -> void:
 	var velocity_clamped = clamp(velocity.length(), 0.5, SPRINT_SPEED * 2)
 	var target_fov = BASE_FOV + FOV_CHANGE * velocity_clamped
 	camera.fov = lerp(camera.fov, target_fov, delta * 8.0)
-
+	
 	move_and_slide()
 
-func _headbob(time) -> Vector3:
+
+func _headbob(time) -> Vector3 :
 	var pos = Vector3.ZERO
 	pos.y = sin(time * BOB_FREQ) * BOB_AMP
 	pos.x = cos(time * BOB_FREQ / 2) * BOB_AMP
 	return pos
+
+
+func health_tracker(max_damage) -> void :
```

Comment:
Consider naming functions as verbs not nouns, so `track_health(damage)` or maybe something like `take_damage(amount)` could be better. Especially when you consider calling the function from some other script: `player.take_damage(10)` vs `player.health_tracker(10)`.
Also, your parameter name is `max_damage` which I think could be misleading. What's max about it?

Reply:
Good shout. Verbs not nouns. I can remember that! Suggested changes implemented and checked